packages/vuetify/src/mixins/activatable/index.ts

```diff
@@ -119,18 +127,53 @@ export default baseMixins.extend({
       if (this.activator) {
         const target = this.internalActivator ? this.$el : document
 
-        activator = typeof this.activator === 'string'
-          ? target.querySelector(this.activator)
-          : this.activator
+        // #foobar
+        if (typeof this.activator === 'string') {
+          activator = target.querySelector(this.activator)
+        // VNode
+        } else if ((this.activator as any).$el) {
+          activator = (this.activator as any).$el
+        // HTMLElement | Element
+        } else {
+          activator = this.activator
+        }
       } else if (e) {
-        activator = e.currentTarget || e.target
+        activator = (e.currentTarget || e.target) as HTMLElement
       } else if (this.activatorNode.length) {
-        activator = this.activatorNode[0].elm
+        activator = this.activatorNode[0].elm as HTMLElement
       }
 
-      this.activatorElement = activator as HTMLElement
+      this.activatorElement = activator
 
       return this.activatorElement
     },
+    getContentSlot () {
+      return getSlot(this, 'default', this.getValueProxy(), true)
+    },
+    getValueProxy (): object {
+      const self = this
+      return {
+        get value () {
+          return self.isActive
+        },
+        set value (isActive: boolean) {
+          self.isActive = isActive
+        },
+      }
+    },
+    removeActivatorEvents () {
+      if (
+        !this.activator ||
+        !this.activatorElement
+      ) return
+
+      for (const event of this.events) {
+        (this.activatorElement as any)[`on${event}`] = null
```

Comment:
removeEventListener?

Reply:
There was a specific reason for this, I'll double check as to why.